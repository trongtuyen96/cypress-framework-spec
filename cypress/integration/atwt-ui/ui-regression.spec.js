describe('UI Regression test', () => {
    it('Compare banner image of homepage', () => {
        cy.visit('/');
        cy.xpath('//div[@data-testid="container-bg"]').toMatchSnapshot();
    })

    it.skip('Compare banner image of post "Overriding vs Overloading"', () => {
        cy.visit('post/overriding-vs-overloading');
        cy.xpath('//img[contains(@style,"object-position: 50% 50%")]').toMatchSnapshot();
    })

    it.skip('Compare banner image of post "SOLID"', () => {
        cy.visit('post/s-o-l-i-d-the-first-5-principles-of-object-oriented-programing');
        cy.xpath('//img[contains(@style,"object-position: 50% 50%")]').toMatchSnapshot();
    })

    it.skip('Compare banner image of post "Java Reflection"', () => {
        cy.visit('post/java-reflection');
        cy.xpath('//img[contains(@style,"object-position: 50% 50%")]').toMatchSnapshot();
    })
})