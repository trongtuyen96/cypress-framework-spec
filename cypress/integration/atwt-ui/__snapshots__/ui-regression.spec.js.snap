exports[`UI Regression test > Compare banner image of homepage #0`] = `
<div class="qDqoW" data-testid="container-bg"></div>
`;

exports[`UI Regression test > Compare banner image of post "Overriding vs Overloading" #0`] = `
<img
  src="https://static.wixstatic.com/media/850ae7_f6cdf44eec0a4375b79cb7a5ed9cb20c~mv2.png/v1/fill/w_81,h_51,al_c,q_85,usm_0.66_1.00_0.01,blur_2,enc_auto/850ae7_f6cdf44eec0a4375b79cb7a5ed9cb20c~mv2.png"
  alt=""
  style="width: 740px; height: 466px; object-fit: cover; object-position: 50% 50%;"
  data-pin-url="https://www.automatedtestingwithtuyen.com/post/overriding-vs-overloading"
  data-pin-media="https://static.wixstatic.com/media/850ae7_f6cdf44eec0a4375b79cb7a5ed9cb20c~mv2.png/v1/fit/w_805%2Ch_507%2Cal_c,enc_auto/file.png"
/>
`;

exports[`UI Regression test > Compare banner image of post "SOLID" #0`] = `
<img
  src="https://static.wixstatic.com/media/850ae7_43463c0f4bfb4e45b2023a34af3cd04c~mv2.png/v1/fill/w_83,h_52,al_c,q_85,usm_0.66_1.00_0.01,blur_2,enc_auto/850ae7_43463c0f4bfb4e45b2023a34af3cd04c~mv2.png"
  alt=""
  style="width: 740px; height: 464px; object-fit: cover; object-position: 50% 50%;"
  data-pin-url="https://www.automatedtestingwithtuyen.com/post/s-o-l-i-d-the-first-5-principles-of-object-oriented-programing"
  data-pin-media="https://static.wixstatic.com/media/850ae7_43463c0f4bfb4e45b2023a34af3cd04c~mv2.png/v1/fit/w_832%2Ch_522%2Cal_c,enc_auto/file.png"
  data-load-done=""
/>
`;

exports[`UI Regression test > Compare banner image of post "Java Reflection" #0`] = `
<img
  src="https://static.wixstatic.com/media/850ae7_1adcfe45e19248a5be95a8b5069ab219~mv2.png/v1/fill/w_83,h_52,al_c,q_85,usm_0.66_1.00_0.01,blur_2,enc_auto/850ae7_1adcfe45e19248a5be95a8b5069ab219~mv2.png"
  alt=""
  style="width: 740px; height: 466px; object-fit: cover; object-position: 50% 50%;"
  data-pin-url="https://www.automatedtestingwithtuyen.com/post/java-reflection"
  data-pin-media="https://static.wixstatic.com/media/850ae7_1adcfe45e19248a5be95a8b5069ab219~mv2.png/v1/fit/w_830%2Ch_523%2Cal_c,enc_auto/file.png"
  data-load-done=""
/>
`;
